### ISSUE TITLE
release/4.6.0 OperaBlink built manifest not populating version

### ISSUE THREAD

**Issue by andytuba** [honestbleeps/Reddit-Enhancement-Suite#2601]
```bash
git checkout release/4.6.0
gulp clean; gulp build -o oblink
grep version dist/oblink/manifest.json
```

Why is it still 900.1? It looks the original copy of manifest.json, not the transformed version, is getting saved to `dist/oblink`

**Comment by andytuba** [comment]
We don't need to fix the gulpfile, but @honestbleeps manually fix the version before submitting to the store.

**Comment by erikdesjardins** [comment]
Gets overridden because we still copy `OperaBlink/*.json` [here](https://github.com/honestbleeps/Reddit-Enhancement-Suite/blob/b0a3275ac1d0ae2b06fbde8085f1946df924a777/gulpfile.js#L81).

**Comment by andytuba** [comment]
I suppose that's worth fixing for 4.6.1 at least.

**Comment by andytuba** [comment]
Thanks for pointing to that!

**Comment by erikdesjardins** [comment]
Fixed for 4.7.0 (by virtue of there not being a separate OperaBlink version)
